src/test/kotlin/io/github/nomisrev/routes/TagRouteSpec.kt

```diff
@@ -10,13 +10,9 @@ import io.github.nomisrev.withServer
 import io.kotest.assertions.arrow.core.shouldBeRight
 import io.kotest.assertions.arrow.core.shouldBeSome
 import io.kotest.core.spec.style.StringSpec
-import io.kotest.matchers.collections.shouldHaveSize
-import io.kotest.matchers.shouldBe
-import io.ktor.client.call.body
-import io.ktor.client.plugins.resources.get
-import io.ktor.http.ContentType
-import io.ktor.http.HttpStatusCode
-import io.ktor.http.contentType
+import io.ktor.client.call.*
+import io.ktor.client.plugins.resources.*
+import io.ktor.http.*
```

Comment:
I don't use _star_ imports in my projects. Maybe I should add Detekt to check for these cases 🤔

Reply:
I also don't use star imports. Sorry for missing it.